recipes/ipyvuetify/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "ipyvuetify" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ece48f57f8ead8ca5dde2f53528d843f557ab2b5d18351636623e662b0d387e0
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
```

This needs dependencies, i think in setup.py we only have ipywidgets
```suggestion
   - ipywidgets >=7.0
```